src/rt/sections_ldc.d

```diff
@@ -426,8 +426,6 @@ private
 {
     version (OSX)
     {
-        extern(C) void _d_dyld_getTLSRange(void*, void**, size_t*);
-        private align(16) ubyte dummyTlsSymbol = 42;
         // By initalizing dummyTlsSymbol with something non-zero and aligning
         // to 16-bytes, section __thread_data will be aligned as a workaround
         // for https://github.com/ldc-developers/ldc/issues/1252
```

Comment:
What about this comment?

Reply:
> What about this comment?

Ah, I hadn't noticed it refers to the removed var. Good catch.

Reply:
Fixed.